test/framework/src/main/java/org/elasticsearch/transport/AbstractSimpleTransportTestCase.java

```diff
@@ -895,13 +895,12 @@ protected void doRun() throws Exception {
                         final String info = sender + "_B_" + iter;
                         serviceB.sendRequest(
                             nodeA,
-                            "test",
+                            "internal:test",
                             new TestRequest(info),
                             new ActionListenerResponseHandler<>(listener, TestResponse::new)
                         );
                         try {
                             listener.actionGet();
-
                         } catch (Exception e) {
                             logger.trace(
```

Comment:
Should we add an `assert false` here too?

Reply:
Mmmmaybe; I didn't spend much time reverse-engineering what exactly this test is trying to do, but I think we can sometimes get errors from our actions since we restart and disconnect things concurrently.